src/jdk.crypto.cryptoki/share/classes/sun/security/pkcs11/SunPKCS11.java

```diff
@@ -521,18 +526,29 @@ private static int[] m(long m1, long m2, long m3, long m4) {
 
     private static void d(String type, String algorithm, String className,
             int[] m) {
-        register(new Descriptor(type, algorithm, className, null, m));
+        register(new Descriptor(type, algorithm, className, null, m, null));
```

Comment:
Would it not just make sense to add a wrapper constructor that passes `null` to the new version? That would simplify this code and avoid a repeat of the issue we had when we changed a signature before.

Reply:
I'll apply this suggestion to the patch for OpenJDK upstream. Thanks.